pkgs/development/tools/mix2nix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, elixir, erlang }:
+
+stdenv.mkDerivation {
+  pname = "mix2nix";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "ydlr";
+    repo = "mix2nix";
+    rev = "66c277bdffb93da9a63e3f1ea42432a73e572b51";
```

Comment:
Should we do `let version = "0.1.3"; in ...` or `mkDerivation rec { ... }` and do `rev = version;` here?

Reply:
Absolutely. Thanks!